src/main/java/ca/corefacility/bioinformatics/irida/processing/impl/FastqcFileProcessor.java

```diff
@@ -92,7 +97,7 @@ private void processSingleFile(SequenceFile sequenceFile) throws FileProcessorEx
 						LocaleContextHolder.getLocale()));
 		try {
 			uk.ac.babraham.FastQC.Sequence.SequenceFile fastQCSequenceFile = SequenceFactory.getSequenceFile(
-					fileToProcess.toFile());
+					sequenceFile.getTemporaryFile());
```

Comment:
Is this the right logic here?  Now `fileToProcess` isn't getting used.

Reply:
That is correct. But if we want to keep the getTemporaryFile in the iridaFileStorageUtility classes then we will need to use it

Reply:
Yeah understood.  We'll have to finish that conversation first.

Reply:
Updated in 2f3e957f7a3a515362d5c8c107008236a1146e4d